internal/host/stopinstallation.go

```diff
@@ -10,21 +14,33 @@ import (
 
 type stopInstallationCmd struct {
 	baseCmd
+	instructionConfig InstructionConfig
 }
 
-func NewStopInstallationCmd(log logrus.FieldLogger) *stopInstallationCmd {
+func NewStopInstallationCmd(log logrus.FieldLogger, instructionConfig InstructionConfig) *stopInstallationCmd {
 	return &stopInstallationCmd{
-		baseCmd: baseCmd{log: log},
+		baseCmd:           baseCmd{log: log},
+		instructionConfig: instructionConfig,
 	}
 }
 
 func (h *stopInstallationCmd) GetStep(ctx context.Context, host *models.Host) (*models.Step, error) {
-	step := &models.Step{
-		StepType: models.StepTypeExecute,
-		Command:  "/usr/bin/podman",
-		Args: []string{
-			"kill", "--all",
-		},
+	step := &models.Step{}
+	step.StepType = models.StepTypeExecute
+	step.Command = "/usr/bin/podman"
+	cmdArgs := []string{"kill", "--all"}
+
+	// added to run upload logs if we are in error or cancelled state. Stop all and gather logs
+	// will return same exit code as stop command command
```

Comment:
Actually that isn't really required. 99.9% of the time this command fail since it has nothing to kill.
Let's remove this logic make the gather logs command simple.

Reply:
ok

Reply:
Ok....?